universal-application-tool-0.0.1/conf/routes

```diff
@@ -49,8 +49,12 @@ GET     /admin/questions/:id/translations/:locale/edit  controllers.admin.AdminQ
 POST    /admin/questions/:id/translations/:locale       controllers.admin.AdminQuestionTranslationsController.update(request: Request, id: Long, locale: String)
 
 # Trusted Intermediary routes
-GET     /admin/tis               controllers.admin.TrustedIntermediaryManagementController.index(request: Request)
-POST    /admin/tis/new           controllers.admin.TrustedIntermediaryManagementController.create(request: Request)
+GET     /admin/tis                    controllers.admin.TrustedIntermediaryManagementController.index(request: Request)
```

Comment:
(blocking) the resource here is a `trusted intermediary group` right? I think `tis` is not descriptive enough. It should at least be `tiGroup`, if not the whole thing.